src/bentoml/_internal/cloud/__init__.py

```diff
@@ -1,4 +1,3 @@
 from .base import CloudClient as CloudClient
 from .bentocloud import BentoCloudClient as BentoCloudClient
-from .deployment import Resource as Resource
```

Comment:
Technically this is a breaking change.

Probably should throw deprecation warning either way a getattr

Reply:
Actually this is a class so I don't think we can make getattr warning 🤔

Reply:
Thanks, this is fixed